src/Todo/Todo.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from '../User/User';
+
+function Todo({ todo: { title, user, completed } }) {
+  return (
+    <>
+      <User user={user} />
```

Comment:
```suggestion
      <User {...user} />
```
три слова user подряд не добавляют коду читабельности